resources/roca_main.brs

```diff
@@ -26,9 +24,31 @@ sub main()
     }
 
     for each file in files
-        path = ["pkg:", basePath, file].join("/")
-        suite = _brs_.runInScope(path, args)
+        suite = _brs_.runInScope(file, args)
         args.index += 1
     end for
 end sub
 
+' Recursively searches directories for '*.test.brs' files starting at `path`.
+'
+' @param {string} [path="pkg:"] - the path to search
+' @param {roArray} [testFiles=[]] - the current set of discovered test files
+'
+' @returns {roArray} an array containing the fully-qualified path to each '*.test.brs' file
+'                    discovered recursively from `path`
+function __roca_findTestFiles(path = "pkg:" as string, testFiles = [] as object) as object
+    files = ListDir(path)
+
+    for each f in files
```

Comment:
nit: can we just call this `file` instead of `f`? This is consistent with other parts of this file.

Reply:
Whoops, you're right!  That's what I get for trying to write this in a cramped hospital recovery room 😅